test/.jshintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../.jshintrc",
+  "maxlen": null,
```

Comment:
Or I can actually fix the violations. A few were names of tests though, might be weird with a line-break in them?